src/localization/en.js

```diff
@@ -153,6 +153,9 @@ export default {
     },
 
     currentIncome: {
+      title:        'Current Household Income',
+      clarifier:    'Income that you collected in the past 12 months.',
```

Comment:
This and the line above are incorrectly indented and will create a merge conflict unless fixed before the next PR. You can probably just run `npm run lint`.